scripts/setup-flex-builddir

```diff
@@ -261,6 +261,7 @@ setup_builddir () {
     case ${MACHINE} in
         qemu*)
             OPTIONALLAYERS="$OPTIONALLAYERS flex-bsp-$MACHINE"
+            EXCLUDEDLAYERS="flex-vendor-integration"
```

Comment:
this will exclude if we will be including the layer in layer.conf. I think commit message needs to be updated. it remove the layer from setup

Reply:
@Noor-Ahsan this will exclude the vendor-integration layer when we use the setup-env script to create the build configuration. Let me add further description to the commit message.

Reply:
@tarmeh2r nitpick here. Please don't override excluded layers rather append to it. i.e.
```
EXCLUDEDLAYERS="$EXCLUDEDLAYERS flex-vendor-integration"
```

Reply:
Thanks @abelal I have made the requested changes. @Noor-Ahsan I have also added further explanation in the commit message.